examples/tailwind-css/tailwind.config.js

```diff
@@ -4,5 +4,8 @@ module.exports = {
   theme: {
     extend: {},
   },
+  corePlugins: {
+    preflight: false,
```

Comment:
This could help developers that start from the example, and 6 months later on, wonder why the property is for.

```suggestion
    // Remove Tailwind CSS's preflight style as it applies global change
    // that break MUI components, we replace it with CssBaseline.
    preflight: false,
```